source/_patterns/02-organisms/components/hero-banner~interview.json

```diff
@@ -0,0 +1,30 @@
+{
+  "image": {
+    "sources": [
+      {
+        "srcset": "https://iiif.elifesciences.org/lax/09560%2Felife-09560-fig1-v1.tif/full/full/0/default.jpg",
+        "type": "image/jpeg"
+      }
+    ],
+    "fallback": {
+      "defaultPath": "https://unsplash.it/1114/336",
+      "altText": ""
+    }
+  },
+  "title": "Controlling traffic",
+  "titleLength": "large",
+  "url": "#",
+  "summary": "Ramanath Hegde is a Postdoctoral Fellow at the Institute of Protein Biochemistry in Naples, Italy, where he investigates ways of preventing cells from destroying mutant proteins.",
+  "meta": {
+    "url": "#",
+    "text": "Interview",
+    "date": {
+      "forHuman": {
+        "dayOfMonth": 29,
+        "month": "Jan",
+        "year": 2016
+      },
+      "forMachine": "2016-01-29"
+    }
+  }
+}
```

Comment:
nitpick: any file should have an empty line at the end of the file

Reply:
Empty line is added to all json files